packages/office-ui-fabric-react/src/index.ts

```diff
@@ -8,7 +8,7 @@ export * from './Callout';
 export * from './Check';
 export * from './Checkbox';
 export * from './ChoiceGroup';
-export * from './ChoiceGroupOption';
+// export * from './ChoiceGroupOption'; // exported by ChoiceGroup
```

Comment:
Do we need to leave this commented in here? Can we just remove it?

Reply:
```suggestion
```

Reply:
just remove it?

Reply:
I want to leave it as documentation.

Reply:
I think it would be cleaner to add `// ChoiceGroupOption is also exported by ChoiceGroup` to end of line 10, and remove line 11 entirely..